### PR TITLE
Tidy up test and skip if LWP::ConsoleLogger::Easy < 0.000028.

### PR DESCRIPTION
**Pull request by manwar**
Hi @oalders 

Please review the PR.
It address the following issue:

     manwar@ubuntu:~/github/webservice-sqlformat$ make test
     PERL_DL_NONLAZY=1 "/usr/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" 
     "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')
     " t/*.t t/WebService/*.t
     t/WebService/SQLFormat.t .. 1/? LWP::ConsoleLogger::Easy version 0.000028 required--this is 
     only version 0.000023 at /usr/local/share/perl/5.18.2/Module/Runtime.pm line 349, <DATA> line 1.
     # Tests were run but no plan was declared and done_testing() was not seen.
     # Looks like your test exited with 255 after test #2.
     t/WebService/SQLFormat.t .. Dubious, test returned 255 (wstat 65280, 0xff00)
     All 2 subtests passed 

     Test Summary Report
     -------------------
     t/WebService/SQLFormat.t (Wstat: 65280 Tests: 2 Failed: 0)
     Non-zero exit status: 255
     Parse errors: No plan found in TAP output
     Files=1, Tests=2,  1 wallclock secs ( 0.05 usr  0.01 sys +  0.58 cusr  0.22 csys =  0.86 CPU)
     Result: FAIL
     Failed 1/1 test programs. 0/2 subtests failed.
     make: *** [test_dynamic] Error 255

Many Thanks.
Best Regards,
Mohammad S Anwar

### CODE REVIEW

File: t/WebService/SQLFormat.t [hunk 1]
```diff
@@ -6,15 +6,21 @@ use Test2::Bundle::More;
 
 use WebService::SQLFormat;
 
-my $f = WebService::SQLFormat->new( debug_level => $ENV{AUTHOR_TESTING}
+SKIP: {
+    my $min_version = '0.000028';
+    eval "use LWP::ConsoeLogger::Easy $min_version";
+    skip "LWP::ConsoleLogger::Easy $min_version required", 3  if $@;
```
Comment (oalders) [inline t/WebService/SQLFormat.t]: What do you think about using https://metacpan.org/pod/Test::Needs here instead? That's a bit less code and saves one level of indentation.

**Review by oalders — CHANGES_REQUESTED** [review]

**Comment by oalders** [comment]
Thanks for this!  I just had a small suggestion.  :)

**Comment by manwar** [comment]
Just pushed the changes as suggested.
I have learnt new thing today. Thanks for sharing your knowledge.

**Review by oalders — APPROVED** [review]

**Comment by oalders** [comment]
> I have learnt new thing today. Thanks for sharing your knowledge.

I'm happy to hear it.  😄 Thanks again!

**Pull request merged**